src/components/Newsletter/Newsletter.js

```diff
@@ -62,7 +64,7 @@ class Newsletter extends React.Component {
       // Merge the locations with any existing ones since that's _probably_ what the user wants.
       await db
         .collection('alerts-subscriptions')
-        .doc(email)
+        .doc(email.toLocaleLowerCase())
```

Comment:
i think we should lowercase all the emails in our db

Reply:
👍